metrics/sinks/hawkular/client.go

```diff
@@ -124,6 +136,20 @@ func (self *hawkularSink) idName(ms *core.MetricSet, metricName string) string {
 	return strings.Join(n, separator)
 }
 
+func (self *hawkularSink) nodeName(ms *core.MetricSet) string {
+	if &self.labelNodeId != nil {
```

Comment:
Would prefer to flip this check & `return ms.Labels[core.LabelNodename.Key]` - no need for the else part, reduces nesting.